README.md

```diff
@@ -53,6 +53,10 @@ Add the port number in the newly created user rc file. You should at least edit
 
     export SERVER_PORT=9000
 
+Add the API key to retrieve departure information from https://opentransportdata.swiss into the user rc file (keepass). For instance:
+
+    export OPENTRANS_API_KEY=dasffdjfjfjfjf566776jfjfjfj22243eac841ce7c9426355b
```

Comment:
Please remove any valid key from the doc. Just put a dummy one. Is it the case here?

Reply:
it is a dummy key ;-)